test/Microsoft.AspNet.Mvc.IntegrationTests/BinderTypeBasedModelBinderIntegrationTest.cs

```diff
@@ -282,6 +288,10 @@ private class SuccessModelBinder : IModelBinder
             public Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContext)
             {
                 var model = "Success";
+                bindingContext.ModelState.SetModelValue(
+                    bindingContext.ModelName,
+                    new ValueProviderResult(model, model, System.Globalization.CultureInfo.CurrentCulture));
```

Comment:
Please add using for `System.Globalization.CultureInfo.CurrentCulture` - there might be a few other places as well.